demo/test_client.html

```diff
@@ -20,28 +20,45 @@ <h2>fxa-pairing-tls demo client</h2>
 
   <script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
   <script src="../dist/fxaPairingTLS.js"></script>
-  <script src="./tlschannel.js"></script>
   <script>
     $(() => {
+      const {InsecurePairingChannel} = fxaPairingTLS;
```

Comment:
we should rename this in the future

Reply:
Yeah, I wonder I we should rename the entire repo to "pairing-channel" rather than "pairing-tls" in order to put the emphasis on the public API rather than the crypto details.